counterpoint/lib/attribute.ts

```diff
@@ -684,7 +684,15 @@ export class Attribute<
     this.valueFn = undefined;
     return this;
   }
-
+  /**
+   * Sets transform of an attribute to a new function, can be applied to an Attribute,
+   * a Mark, or a MarkRenderGroup
+   * @param attrToModify attribute to modify
+   * @param newFunc new function to set transform to
+   */
+  setTransform(attrToModify: Attribute<TransformedValueType, ValueType, ComputeArgumentType> = this, newFunc : (raw : ValueType, computeArg : ComputeArgumentType) => TransformedValueType) : void {
```

Comment:
make sure to run the Prettier code formatter so that we don't have really long lines